androidplot-core/src/main/java/com/androidplot/xy/PanZoom.java

```diff
@@ -95,7 +111,12 @@ public static PanZoom attach(XYPlot plot) {
     }
 
     public static PanZoom attach(XYPlot plot, Pan pan, Zoom zoom) {
-        PanZoom pz = new PanZoom(plot, pan, zoom);
+        return attach(plot,pan,zoom, ZoomLimit.OUTER);
+    }
+
+    // additional attach function not to break api
+    public static PanZoom attach(XYPlot plot, Pan pan, Zoom zoom, ZoomLimit limit) {
```

Comment:
Needs a javadoc describing usage of the additional param.  I'm happy to add these myself if it helps but I didn't want to just do it and knock your branch out of sync with this PR branch